src/test/kotlin/org/malachite/estella/organization/OrganizationIntegration.kt

```diff
@@ -87,6 +88,37 @@ class OrganizationIntegration : BaseIntegration() {
 
     @Test
     @Order(6)
+    fun `should return ok with list of offers`() {
+        val response = getOrganizationsOffers(legitOrganizationUser.mail, legitOrganizationPassword)
+        response.forEach{
+            it.let {
+                expectThat(it.organization.name).isEqualTo(legitOrganizationName)
```

Comment:
Nie wiem, jaka będzie różnica, ale wszystko to możesz zawrzeć w funkcji wyższego rzędu expect (jest zdefiniowana) i zamiast expectThat korzystać z that. Ogólnie wydaje mi się, że będzie czytelniejsze, ale to już up to you

Reply:
@Davenury ale jaka jest różnica pomiędzy napisaniem
```kotlin
 expect{
   that(coś).isEqualTo(coś)
   that(coś.coś).isEqualTo(coś.coś)
 }
```
a, tym
```kotlin
expectThat(coś).isEqualTo(coś)
expectThat(coś.coś).isEqualTo(coś.coś)
```
Bo jak na moje są bardzo podobne tyle tylko że drugie ma mniejsze zagnieżdzenie?

Reply:
Znaczy mi chodziło o coś takiego:
```kotlin
expect {
  response.forEach{
    it.let{ ... }
  }
}
```
Tu wiesz wtedy, gdzie się zaczyna asercja